run_local_outline.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
This line hasn't been necessary for the other .sh files - is there a reason it's added here?